From: Igni <igni@example.com>
Subject: [PATCH] Use direct GitHub URL for OouraFFT instead of mirror

Replace the mirror.tensorflow.org URL with direct GitHub URL for
downloading OouraFFT v1.0.

Signed-off-by: Igni <igni@example.com>
---
 tensorflow/lite/tools/cmake/modules/fft2d.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tensorflow/lite/tools/cmake/modules/fft2d.cmake b/tensorflow/lite/tools/cmake/modules/fft2d.cmake
index 1234567..abcdefg 100644
--- a/tensorflow/lite/tools/cmake/modules/fft2d.cmake
+++ b/tensorflow/lite/tools/cmake/modules/fft2d.cmake
@@ -21,7 +21,7 @@ if(TARGET fft2d OR fft2d_POPULATED)

 OverridableFetchContent_Declare(
   fft2d
-  URL https://storage.googleapis.com/mirror.tensorflow.org/github.com/petewarden/OouraFFT/archive/v1.0.tar.gz
+  URL https://github.com/petewarden/OouraFFT/archive/refs/tags/v1.0.tar.gz
   # Sync with tensorflow/workspace2.bzl
   URL_HASH SHA256=5f4dabc2ae21e1f537425d58a49cdca1c49ea11db0d6271e2a4b27e9697548eb
   SOURCE_DIR "${CMAKE_BINARY_DIR}/fft2d"
